command-expansion-server/test/utils/MissionModel.ts

```diff
@@ -6,11 +6,11 @@ export async function uploadMissionModel(graphqlClient: GraphQLClient): Promise<
   /*
    * Insert a mission model
    */
-  const bananaNationMissionModelUrl = new URL('../inputs/banananation-0.11.0-SNAPSHOT.jar', import.meta.url);
+  const bananaNationMissionModelUrl = new URL('../inputs/banananation-0.11.1-SNAPSHOT.jar', import.meta.url);
```

Comment:
I think this needs to match the jar filename in the inputs directory.

Reply:
https://github.com/NASA-AMMOS/aerie/tree/b03c8c87c95eee56b2baaca1c313a4fc14bd134f/command-expansion-server/test/inputs

Reply:
Thanks for the heads up on this. I've also created a ticket to remove this and grab the jar from the bananation build directory.